### DISCUSSION TITLE
How to calculate exact molecular weight using RdKit Cartridge?

### DISCUSSION THREAD

**Discussion by dzleidig** [rdkit/rdkit#4385] [Q&A]
Is this supported in the cartridge? I am surprised if it is not, but I can't seem to find it anywhere. _Average_ molecular weight is available:
```
select mol_amw('CCC')
> `44.097`
```

It's available in RdKit python:
```
ExactMolWt(Chem.MolFromSmiles('CC'))
30.04...
ExactMolWt(Chem.MolFromSmiles('[13CH3]C'))
31.05...
```

Thanks, 
Dylan

**Comment by greglandrum** [comment]
Looks like exact molecular weight is not available in the cartridge. 
That's an oversight which I will fix for the next release.

**Reply by felipe-imanishi** [reply]
Do you have a timeline for the next release? Also, thank you for the super quick response!

**Reply by dzleidig** [reply]
Thanks, @greglandrum, much appreciated.

**Reply by greglandrum** [reply]
RDKit releases are pretty predictable: there are two feature releases per year - one in March/April and one in September/October - and then patch releases every 4-6 weeks.
This is a new feature and will be in the next feature release.
